### PR TITLE
RSP-1964: Added accessibility statement

### PR DESCRIPTION
**Pull request by jerotire**
## Description

Add an accessibility statement to the Road Side Payments application.

Related issue: [RSP-1964](https://jira.dvsacloud.uk/browse/RSP-1964)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [X] I have performed a self-review of my own code
- [X] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: src/server/i18n/en.json [hunk 1]
```diff
@@ -14,7 +14,8 @@
     "payment": "Payment",
     "active_session": "Active session warning",
     "cookie_preferences": "Cookie preferences",
-    "cookie_details": "Cookie details"
+    "cookie_details": "Cookie details",
+    "accessibility_statement": "Accessibility statement"
```
Comment (grjacob) [inline src/server/i18n/en.json:18]: When another language is being used, will it automatically fall back to the English text?

Comment (jerotire) [reply src/server/i18n/en.json:18]: This is resolved with https://github.com/dvsa/rsp-public-portal/pull/133/commits/acec263d9ba1a00d14ad45ddf47b284772ef11d5

**Review by grjacob — APPROVED** [review]

**Review by mgoldson — APPROVED** [review]
LGTM!

**Review by sdh100shaun — APPROVED** [review]
Looks good -

**Pull request merged**